src/lib/arrays.js

```diff
@@ -0,0 +1,107 @@
+const getNthElement = (index, array) => {
+  if (index < array.length) {
+    return array[index];
+  }
+  return array[index - array.length];
+};
+
+const arrayToCSVString = array => {
+//  console.log(array.toString());
+  return array.toString();
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+function bob(bsr) {
+  return bsr;
+}
+
+const addToArray2 = (element, array) => {
+  const newArray = array.map(bob);
+  newArray.push(element);
+  console.log(newArray);
```

Comment:
The console log should be deleted.
The helper function could be placed inside the addToArray2 function.
You could also refactor this into one line :) by adding the push at the end of the map(bob) and return straightaway.
Or there is a simpler solution, to use the spread operator:
````
  return [...array, element];
`````